internal/tools/pipeline/pipengine/actionexecutor/plugins/k8sjob/k8sjob.go

```diff
@@ -486,6 +487,7 @@ func (k *K8sJob) generateKubeJob(specObj interface{}, clusterInfo apistructs.Clu
 					Volumes:               vols,
 					EnableServiceLinks:    func(enable bool) *bool { return &enable }(false),
 					ShareProcessNamespace: func(b bool) *bool { return &b }(false),
+					HostNetwork:           isHostNetwork,
```

Comment:
kubeJob path: Spec.Template.Spec.DNSPolicy

Need to be set corev1.DNSClusterFirstWithHostNet when `HostNetwork` is `true`. @chengjoey

Reply:
> kubeJob path: Spec.Template.Spec.DNSPolicy
> 
> Need to be set corev1.DNSClusterFirstWithHostNet when `HostNetwork` is `true`. @chengjoey

has DNSPolicy set to `corev1.DNSClusterFirstWithHostNet` been successful for the cdp task?
If the verification is successful, I will submit the changes in this pr